src/shared/sam/debugger/awsSamDebugConfiguration.ts

```diff
@@ -130,14 +134,13 @@ export function createTemplateAwsSamDebugConfig(
             lambda:
                 preloadedConfig.environmentVariables || preloadedConfig.eventJson
                     ? {
-                          event: preloadedConfig.eventJson
-                              ? {
-                                    json: preloadedConfig.eventJson,
-                                }
-                              : undefined,
+                          event: preloadedConfig.eventJson ? { json: preloadedConfig.eventJson } : {},
                           environmentVariables: preloadedConfig.environmentVariables,
                       }
-                    : undefined,
+                    : {
+                          event: {},
```

Comment:
This fallback logic seems to be in a couple of places. Would it be reasonable to centralize that into a method that each place calls into, to set these fields if missing instead?

Reply:
Would the extra indirection increase or decrease clarity?

Reply:
Unknown. I'm thinking about the "next" field that needs this treatment, since this change handles two fields.